client/elements/addons/sc-top-sheet-views.js

```diff
@@ -369,6 +369,28 @@ class SCTopSheet extends LitLocalized(LitElement) {
     this._showToast(this.localizeEx('textViewEnabled', 'textView', this.localize(e.target.dataset.type)));
   }
 
+  get noteDisplayTypeTemplate() {
+    return this.noteDisplayTypeArray.length ? html`
+      <div class="tools">
+        <details><summary>${this.localize('noteSummary')}</summary>
+        <p>${unsafeHTML(this.localize('noteDescription'))}</p></details>
+        <div class="form-controls">
+          ${this.noteDisplayTypeArray.map(({displayType, displayTypeLabel}) => html`
+            <mwc-formfield label="${this.localize(displayType)}">
+              <mwc-radio
+                name="noteDisplayType"
+                value="${displayType}"
+                data-type="${displayTypeLabel}"
+                ?checked="${this.selectedNoteDisplayType === displayType ? true : false}"
```

Comment:
You can simplify it to `{this.selectedNoteDisplayType === displayType}`

Reply:
too late, it's already merged. I can change it during developing the other ticket but to be honest it should be catch be linting tool like eslint, I've seen the same in the other getters.